### ISSUE TITLE
Windows Defender and Anitvirus thinks that electron app is a virus

### ISSUE THREAD

**Issue by isaxk** [benawad/dogehouse#1376]
So I thought I would download Dogehouse for Windows using the .exe installer and Windows Defender _and_ my Antivirus thought it was a virus. I think this was due to the lack of a certificate or something else like that.

- Just thought I report that!

**Comment by TheOtterlord** [comment]
Yeah, we don't yet have code signing for the windows installer. You should be able to install it by following the instructions for your anti-virus provider and windows defender.

**Comment by goldyydev** [comment]
This is something for @benawad to do, he'll need to get a certificate and sign the code using a signing tool (that Windows accepts).

**Comment by amitojsingh366** [comment]
Not sure if it'll help but here's something that happened to me: 
I had a similar issue with my [own project](https://github.com/amitojsingh366/Flappytoj) where windows defender itself thought my app was a virus. To solve this i didn't add any certificate i just did a [Virus Total](https://www.virustotal.com) scan of my `.exe` and it showed that 1/67 engines detected my app as an virus and that was Microsoft's engine. I think somehow that informed Microsoft about the false positive or im not really sure how it happened but after like a day of me scanning the `.exe` on  [Virus Total](https://www.virustotal.com) Windows defender stopped detecting it as a virus and even Microsoft's engine on  [Virus Total](https://www.virustotal.com) gave it the green flag.

**Comment by isaboo** [comment]
I could install it but the button to install was under the tiny 'more info' button. And more info looked like a link more than a button!

**Comment by isaboo** [comment]
@amitojsingh366 said > Not sure if it'll help but here's something that happened to me: 
> I had a similar issue with my [own project](https://github.com/amitojsingh366/Flappytoj) where windows defender itself thought my app was a virus. To solve this i didn't add any certificate i just did a [Virus Total](https://www.virustotal.com) scan of my `.exe` and it showed that 1/67 engines detected my app as an virus and that was Microsoft's engine. I think somehow that informed Microsoft about the false positive or im not really sure how it happened but after like a day of me scanning the `.exe` on  [Virus Total](https://www.virustotal.com) Windows defender stopped detecting it as a virus and even Microsoft's engine on  [Virus Total](https://www.virustotal.com) gave it the green flag.

Ok, that's good then

**Comment by isaboo** [comment]
> Not sure if it'll help but here's something that happened to me: 
> I had a similar issue with my [own project](https://github.com/amitojsingh366/Flappytoj) where windows defender itself thought my app was a virus. To solve this i didn't add any certificate i just did a [Virus Total](https://www.virustotal.com) scan of my `.exe` and it showed that 1/67 engines detected my app as an virus and that was Microsoft's engine. I think somehow that informed Microsoft about the false positive or im not really sure how it happened but after like a day of me scanning the `.exe` on  [Virus Total](https://www.virustotal.com) Windows defender stopped detecting it as a virus and even Microsoft's engine on  [Virus Total](https://www.virustotal.com) gave it the green flag.

**Comment by isaboo** [comment]
@amitojsingh366 said:
> Not sure if it'll help but here's something that happened to me: 

> I had a similar issue with my [own project](https://github.com/amitojsingh366/Flappytoj) where windows defender itself thought my app was a virus. To solve this i didn't add any certificate i just did a [Virus Total](https://www.virustotal.com) scan of my `.exe` and it showed that 1/67 engines detected my app as an virus and that was Microsoft's engine. I think somehow that informed Microsoft about the false positive or im not really sure how it happened but after like a day of me scanning the `.exe` on  [Virus Total](https://www.virustotal.com) Windows defender stopped detecting it as a virus and even Microsoft's engine on  [Virus Total](https://www.virustotal.com) gave it the green flag.

**Comment by TheOtterlord** [comment]
> I could install it but the button to install was under the tiny 'more info' button. And more info looked like a link more than a button!

Yeah, it's a weird route, but it's how you have to bypass it. As I can't afford code-signing for my side project in electron, I just put up some instructions for allowing it passed windows defender in the docs.

**Comment by isaboo** [comment]
@amitojsingh366 said:
> Not sure if it'll help but here's something that happened to me: 
I had a similar issue with my [own project](https://github.com/amitojsingh366/Flappytoj) where windows defender itself thought my app was a virus. To solve this i didn't add any certificate i just did a [Virus Total](https://www.virustotal.com) scan of my `.exe` and it showed that 1/67 engines detected my app as an virus and that was Microsoft's engine. I think somehow that informed Microsoft about the false positive or im not really sure how it happened but after like a day of me scanning the `.exe` on  [Virus Total](https://www.virustotal.com) Windows defender stopped detecting it as a virus and even Microsoft's engine on  [Virus Total](https://www.virustotal.com) gave it the green flag.

Well that's good then
